docs/modules/develop/pages/guide_development_app.adoc

```diff
@@ -4,14 +4,19 @@ In order to start developing you will need what is called a `development_app`. T
 app (that you can create with `decidim app_name`) but with a Gemfile pre-configured for local development and some other small config modifications.
 
 We recommend that you first have a xref:install:manual.adoc[working Decidim app] so that you have fullfilled all the necessary
-system and services requirements, like having a working Ruby installation, PostgreSQL, Node.JS, etc. For better following this
-guide and as you should be working in a development environment (ie without sensitive data in your database), for better comfort
-you can configure PostgreSQL with a relaxed security. See more information about this in the <<database,Database>> section of this page.
+system and services requirements, like having a working Ruby installation, PostgreSQL, Node.JS, etc.
 
-You need it in order to have a Rails application configured to lookup Decidim modules from your filesystem. This way changes
-in your modules will be directly observed by this `development_app`.
+For better following this guide and as you should be working in a development environment (ie without sensitive data in your database),
+for more comfort you can configure PostgreSQL with a relaxed security. This is not recommended for staging nor production configuration,
+just for local development. These instructions are tested in Ubuntu.
 
-You can create a `development_app` from inside the project's root folder with the command:
+[source,console]
+----
+export DATABASE_HOST=/var/run/postgresql/ >> ~/.bashrc
+sudo su - postgres -c "createuser -s $USER"
```

Comment:
Never seen this before (never needed either), not sure if this leads to more confusion. Where this env '$USER' comes from?
What I would mention here is that you need Postgresql configured in you computer (so `DATABASE_HOST=localhost` by default, no configuration needed). And then let clear that the user needs to set up a username/password with database creation/drop permissions for easy recreation of the development_app database.

The easiest way, no extra requirements, would be:

1. Configure postgresql with a super-user: 
```bash
sudo -u postgres psql -c "CREATE USER decidim_user WITH SUPERUSER CREATEDB NOCREATEROLE PASSWORD 'decidim_password'"
```
2. Export the variables export DATABASE_USERNAME, DATABASE_PASSWORD before using any decidim-rails command. The more versatile way to do this is to use `.rbenv-vars` but for a very basic setup this should suffice:
```bash
echo "export DATABASE_USERNAME=decidim_user" >> ~/.bashrc
echo "export DATABASE_PASSWORD=password_password" >> ~/.bashrc
```

Reply:
Ok, I've changed applied your suggestions changing them a bit. I've also tested them in a new environment so I can confirm it works. How about now?

Reply:
Now we have it with rbenv-vars 😄